### PR TITLE
 Fix Rails 5 warning messages

### LINKED ISSUE #801: Rails 5 warning messages
Rails 5 specs should run without warnings.

See some related code in: https://github.com/thoughtbot/high_voltage/pull/229/files#diff-3b61ca35a7f47da3a2dbf21b5b9e3dd0R1

### PR DESCRIPTION
**Pull request by nbulaj**
Fixes #801: removes deprecation messages, uses a new version of RSpec with Rails 5.

Checked with Rails 5.0.0.beta3 and 5.0.0.rc1.

### CODE REVIEW

File: lib/doorkeeper/orm/active_record.rb [hunk 1]
```diff
@@ -20,11 +20,8 @@ def self.initialize_application_owner!
...
-        if ::ActiveRecord::Base.connected? &&
-           ::ActiveRecord::Base.connection.table_exists?(
-             Doorkeeper::Application.table_name
-           )
-          unless Doorkeeper::Application.column_names.include?("scopes")
+        if ::ActiveRecord::Base.connected? && Doorkeeper::Application.table_exists?
+          if Doorkeeper::Application.column_names.exclude?("scopes")
             migration_path = '../../../generators/doorkeeper/templates/add_scopes_to_oauth_applications.rb'
             puts <<-MSG.squish
 [doorkeeper] Missing column: `oauth_applications.scopes`.
```
Comment (tute) [inline lib/doorkeeper/orm/active_record.rb]: This warning message was useful while upgrading to doorkeeper 3. I'll ditch it altogether for v4.

Comment (tute) [inline lib/doorkeeper/orm/active_record.rb]: > This warning message was useful while upgrading to doorkeeper 3. I'll ditch it altogether for v4.

https://github.com/doorkeeper-gem/doorkeeper/pull/838

Comment (nbulaj) [inline lib/doorkeeper/orm/active_record.rb]: Nice! I love it :)

File: spec/dummy/config/environments/test.rb [hunk 2]
```diff
@@ -8,7 +8,11 @@
   config.cache_classes = true
 
   # Configure static asset server for tests with Cache-Control for performance
-  config.static_cache_control = 'public, max-age=3600'
+  if ::Rails::VERSION::MAJOR >= 5
+    config.public_file_server.headers = { 'Cache-Control' => 'public, max-age=3600' }
+  else
+    config.static_cache_control = 'public, max-age=3600'
+  end
```
Comment (tute) [inline spec/dummy/config/environments/test.rb]: We probably can drop this configuration altogether.

Comment (nbulaj) [inline spec/dummy/config/environments/test.rb]: I check the code and yes, you are right - we can drop that config.

**Comment by tute** [comment]
Merged in #838. Ready for this to be rebased and to drop the test cache configuration. Thank you! 👏

**Comment by tute** [comment]
Also pushed a commit upgrading from beta3 to rc1.

**Comment by nbulaj** [comment]
@tute done.

File: .travis.yml [hunk 3]
```diff
@@ -12,9 +12,9 @@ before_install:
 
 env:
   - rails=4.2.0
-  - rails=5.0.0.beta3
+  - rails=5.0.0.rc1
 
 matrix:
   exclude:
-    - env: rails=5.0.0.beta3
+    - env: rails=5.0.0.rc1
```
Comment (tute) [inline .travis.yml]: I don't see why this appears as a diff if `master` contains this commit: https://github.com/doorkeeper-gem/doorkeeper/commit/eb40ba04518b76bc074865114e46e8bc9f549333.

Also, I look like the author of your rebased history. Do you want to `git commit --amend --reset-author`?

**Comment by tute** [comment]
There is a commit in master that wasn't rebased in. In case it's useful, these are the steps I follow:

```
git remote add upstream https://github.com/doorkeeper-gem/doorkeeper.git
git fetch upstream
git checkout BRANCH
git rebase upstream/master
# Fix conflicts, then `git rebase --continue`, repeat until done
git push --force-with-lease
```

**Comment by tute** [comment]
Merged into master. Thank you! 😃

**Pull request closed without merging**